resharper/resharper-unity/src/Unity/Core/ProjectModel/UnitySolutionTracker.cs

```diff
@@ -41,7 +41,7 @@ public UnitySolutionTracker(ISolution solution, IFileSystemTracker fileSystemTra
         private void SetValues()
         {
             IsUnityProjectFolder.SetValue(HasUnityFileStructure(mySolution.SolutionDirectory));
-            IsUnityProject.SetValue(IsUnityProjectFolder.Value && mySolution.IsValid() && mySolution.SolutionFilePath.ExistsFile);
+            IsUnityProject.SetValue(IsUnityProjectFolder.Value && mySolution.IsValid() && mySolution.SolutionFilePath.ExistsFile && HasLibraryFolder(mySolution.SolutionDirectory));
```

Comment:
Should `HasLibraryFolder` be part of `HasUnityFileStructure`?

Reply:
No, because Library is usually not pushed to vcs. After checking out project from vcs, we need to show notification/action to start unity. After solution is generated, Library folder also already exist.